src/ctia/entity/incident.clj

```diff
@@ -46,6 +53,65 @@
 (def realize-incident
   (default-realize-fn "incident" NewIncident StoredIncident))
 
+(s/defschema IncidentStatus
+  (fs/->schema vocs/Status))
+
+(s/defschema IncidentStatusUpdate
+  {:status IncidentStatus})
+
+(defn make-status-update
+  [{:keys [status]}]
+  (let [t (time/internal-now)
+        verb (case status
+               "New" nil
+               "Stalled" nil
+               ("Containment Achieved"
+                "Restoration Achieved") :remediated
+               "Open" :opened
+               "Rejected" :rejected
+               "Closed" :closed
+               "Incident Reported" :reported
+               nil)]
+    (cond-> {:status status}
+      verb (assoc :incident_time {verb t}))))
+
+(def incident-operation-routes
+  (routes
+   (POST "/:id/status" []
+         :return Incident
+         :body [update IncidentStatusUpdate
+                {:description "an Incident Status Update"}]
+         :header-params [{Authorization :- (s/maybe s/Str) nil}]
+         :summary "Update an Incident Status"
+         :path-params [id :- s/Str]
+         :capabilities :create-incident
+         :auth-identity identity
+         :identity-map identity-map
+         (let [status-update (make-status-update update)]
+           (if-let [updated
+                    (-> (flows/patch-flow
+                         :get-fn #(read-store :incident
+                                              read-record
+                                              %
+                                              identity-map
+                                              {})
+                         :realize-fn realize-incident
```

Comment:
I think that the `status-update` should be done by the `realize-fn`. You could compose the default `realize-incident` with the `status-update`

Reply:
good idea, I might change my mind but I prefer to have it explicitly defined that way, will change it if that becomes an issue.